.mocharc.yml

```diff
@@ -3,4 +3,3 @@ reporter: spec
 ui: bdd
 full-trace: true
 exit: true
-parallel: true
```

Comment:
Any reason why `parallel` was disabled?

Reply:
I've tried keeping parallel, but then tests containing async on my computer throw exceptions, and I don't know why
![image](https://github.com/hexojs/hexo/assets/49871906/0c4eb3b9-168b-4e4c-97df-5f8c141b104c)
![image](https://github.com/hexojs/hexo/assets/49871906/838fcc4d-14bb-4014-8510-cc6340638cd8)

Reply:
parallel makes it easier to make tests timeout in my experience